lib/libvarnishapi/vsc.c

```diff
@@ -319,7 +319,6 @@ vsc_del_seg(const struct vsc *vsc, struct vsm *vsm, struct vsc_seg **spp)
 	struct vsc_seg *sp;
 
 	CHECK_OBJ_NOTNULL(vsc, VSC_MAGIC);
-	AN(vsm);
```

Comment:
Why remove this one?

Reply:
We only pass the "object" so strictly speaking we do not have to check whether it satisfy our need.
Anyways, this should have been `CHECK_OBJ_NOTNULL(vsm, VSM_MAGIC);`, but that is already done by both `VSM_Map`/"VSM_Unmap`.